install.md

```diff
@@ -108,6 +110,7 @@ yum install -y \
   ostree-devel \
   pkgconfig \
   runc \
+  uidmap \
```

Comment:
This should go under the Debian and Ubuntu section not the Fedora/RHeL section no?

Reply:
lol that’s embarrassing :p I’ll fix it up in a few minutes!

Reply:
Fixed.